File: javascript/tempo_de_jogo.js
const horaInicial = parseInt(prompt("Hora inicial"));
const horaFinal = parseInt(prompt("Hora final"));

if (horaFinal > horaInicial) {
    duracao = horaFinal - horaInicial;
} else {
    duracao = horaInicial - (horaFinal + 24);
}

if (duracao < 0) {
    duracao *= -1;
}

console.log("o jogo durou %i hora(s)", duracao);

File: python/tempo_de_jogo.py
hora_inicial = int(input("Hora inicial: "))
hora_final = int(input("Hora final: "))

if hora_final > hora_inicial:
    duracao = hora_final - hora_inicial
else:
    duracao = hora_inicial - (hora_final + 24)

if duracao < 0:
    duracao *= -1

print(f"O JOGO DUROU {duracao} HORA(S)")
